main.cpp

```diff
@@ -68,42 +68,174 @@ struct CarWash
 /*
  1)
  */
+struct guitarRepair
+{
+    struct Guitar  //nested class
+    {
+        int numPickups = 2;
+        int switchPositions = 3;
+        bool needsNewFrets = false;
+    };
+
+    bool upgradesDesired = true;
+    void upgradeGuitar(Guitar guitar);
+    bool checkGuitar(Guitar guitar);
+};
 
 /*
  2)
  */
+struct bassRepair
+{
+    struct Bass
+    {
+        struct strings
+        {
+            bool stringsOld = false;
+            void changeStrings(strings Strings);
+        };
+        struct electronics
+        {
+            bool happyWithElectronics = false;
+        };
+    };
+
+    struct wiringDiagram
+    {
+        bool newPickups = true;
+        int s1Switches = 3;
+        void wireBass(Bass bass);
+    };
+};
 
 /*
  3)
  */
+ struct keyboardRepair
+ {
+     struct Keyboard
+     {
+        bool tinesOkay = false;
+        bool electronicsOkay = false;
+        bool potsOkay = true;
+     };
+     
+     void replaceTines(Keyboard keyboard);
+     void repairElectronics(Keyboard keyboard);
+ };
 
 /*
  4)
  */
+struct speakerRepair
+{
+    struct Speaker
+    {
+        bool voiceCoilOpen = false;
+        bool voiceCoilShort = true;
+        bool tweeterOkay = false;
+    };
+    
+    void replaceDriver(Speaker speaker);
+    void replaceTweeter(Speaker speaker);
+};
 
 /*
  5)
  */
+struct pedalRepair
+{
+    struct Pedal
+    {
+        bool potsOkay = false;
+        bool boardOkay = false;
+        bool wiresBurned = true;
+        bool caseDamaged = true;
+        bool soundOutput = false;
+    };
+    void trash(Pedal myTubeScreamer);
+    void buildNewPedal(Pedal myNewTubeScreamer);
+};
 
 /*
  6)
  */
+struct DIbuild
+{
+    struct Transformer
+    {
+        double ratio = 12;
+        double ratedImpedance = 10000;
+        bool centerTapped = false;
+    };
+    struct Jacks
+    {
+        int XLRJacks = 1;
+        int TSJacks = 1;
+    };
+    void wireTStoPrimary(Jacks DIJacks, Transformer Jensen);
+    void wireXLRtoSecondary(Jacks DIJacks, Transformer Jensen);
+};
 
 /*
- 7)
+ 7) ONLY UDTs
  */
+struct junkyard
+{
+    struct Vehicle;
+    void Demolish(Vehicle Car1);
+    void sell(Vehicle Car2);
+};
 
 /*
  8)
  */
+struct workFromHome
+{
+    bool officeSpaceSetUp = true;
+    int computerMonitors = 3;
+    bool foodinFridge = false;
+    bool workDownloaded = false;
+
+    void goShopping(int itemsOnList = 20);
+    void downloadWork(bool AsperaConnected = true);
+};
 
 /*
  9)
  */
+struct repairPreamp
+{
+    struct Preamp
+    {
+        bool potsOkay = true;
+        bool electronicsOkay = false;
+        double THD = 10;
```

Comment:
You're initializing your `double` with and `int`
- [ ] FIXME: initialize doubles with doubles, not integers or floats.